myhpi/core/management/commands/initializesite.py

```diff
@@ -0,0 +1,18 @@
+from django.core.management.base import BaseCommand
+
+from myhpi.core.models import Footer
+
+
+class Command(BaseCommand):
+    help = "Initializes the website (e.g. creates the default footer)"
+
+    def handle(self, *args, **options):
+        self._add_footer()
+
+    def _add_footer(self):
```

Comment:
we already add the root page in our own migration. I would suggest adding the footer there as well. We can leave this for now and move it to a separate issue